content/global/inclusive.mdx

```diff
@@ -0,0 +1,111 @@
+- Someone escaping from an abusive relationship
+- Someone experiencing ongoing harassment
+- Someone being dogpiled (multiple people taking the same abusive action against you at once, like commenting)
+
+Take notes from this discussion and update any user journeys accordingly.
+
+#### Who might you be excluding?
+
+Another key question to think about beyond abuse is: "Who might you be excluding from partcipating because of the way the feature is built?" When we excitedly jump quickly into building and shipping, it can be easy to forget how our diverse and international community might view our work.
+
+Consider those who are:
+- (Link off to accessibility personas?)
+- Newer to the norms of tech, Software Development, and open source
+- Newer to the Internet
+- Using slower devices
+- On unreliable Internet connections
+- Not fluent in written or spoken English
+
+#### How might we encourage good and positive behavior?
+
+🚧
+
+#### How might we discourage bad or unproductive behavior?
+
+🚧
+
+### Engaging with your target user base
+
+It's critical to build an understanding of the humans you are designing for, whether that's face-to-face engagement or through data. Prepare what data questions you need to better understand the personas you've created in the work sessions above. Whether you're doing exploratory research or validating prototypes, here are some tips for sourcing diverse candidates:
+- Set a goal for who you’d like to represent. Backgrounds to consider:
+  - Location
+  - Experience level in programming/tech
+  - Gender identity
+- Set a balance between solicitng feedback in-product and manually outreaching
+- Need help? Reach out to GitHub employee resource groups (ERGs) to expand your network. Get additional candidate sourcing help from #user-research and #developer-relations, who are very connected to our communities
+
+Once you do find these candidates, **elevate them as your community champions**. Tell their stories internally, credit their work, and keep a feedback loop going with them.
+
+## Fundamentals to include in your design work
+
+These fundamental tools are useful when designing new features, as well as auditing existing ones.
+
+### Scope out minimal viable safety tools
+
+Account for how the feature you’re designing could be used to harm others, and how to mitigate harm in stress cases.
+- [ ] Is there an escape hatch? How easy is it to stop the stress from spreading?
+- [ ] Is there an easy way to report abuse and block the offending user(s)?
+
+### Add trust signals
+
+How might we remind people of the humans behind the code?
+- Usage of avatars where appropriate
+- Usage of full names, not just handles, in workplace environments
+- Mutual collaborators
+- Mutual followers/following
+- Shared languages, contribution, repos, orgs
+- Account age (I know this one is tricky)
```

Comment:
I the text in brackets to be removed? Do you mean how long they've had their account?